packages/playground/_stories/main/Switch/Switch.stories.ts

```diff
@@ -38,30 +38,14 @@ const Template: UI5StoryArgs<Switch, StoryArgsSlots> = (args) => html`<ui5-switc
 ></ui5-switch>`;
 
 export const Basic = Template.bind({});
-
-export const WithText = Template.bind({});
-WithText.args = {
-	textOn: "On",
-	textOff: "Off",
-};
-
-export const Checked = Template.bind({});
-Checked.args = {
-	textOn: "Yes",
-	textOff: "No",
-	checked: true,
+Basic.args = {
+	accessibleName: "Switch with Accessible Name",
 };
 
-export const Disabled = Template.bind({});
-Disabled.args = {
-	disabled: true,
-	checked: true,
-};
-
-export const Design = Template.bind({});
-Design.args = {
+export const Graphical = Template.bind({});
+Graphical.args = {
```

Comment:
This sample is also redundant. The only difference with the basic one is the "graphical" design type. That might be easily changed in the given design type select box.